### PR TITLE
cloudml-template - updating the default params

### PR DESCRIPTION
**Pull request by ksalama**
@puneith - I have made the following (minor) updates to the cloudml-template template & examples:

- Updating the --num-epoch default value to None, since it's use is not relevant unless the --train-size and the --train-batch-size parameters are specified, otherwise the --train-steps is used.

- Simplified the example script to submit a training job to CMLE, including only the necessary arguments

Note that the same exact changes were applied to the template, and the 4 examples.

These changes managed to resolve this issue #203

### CODE REVIEW

**Comment by puneith** [comment]
Thanks @ksalama I have added @dizcology who can help.

File: cloudml-template/examples/babyweight-custom/trainer/task.py [hunk 1]
```diff
@@ -93,7 +93,7 @@ def initialise_hyper_params(args_parser):
         If both --train-size and --num-epochs are specified,
         --train-steps will be: (train-size/train-batch-size) * num-epochs.\
         """,
-        default=10,
+        default=None,
```
Comment (dizcology) [inline cloudml-template/examples/babyweight-custom/trainer/task.py]: This is fine, but please consider further simplifying to allow just one appropriate way of specifying train_steps.  I generally find train_steps and train_batch_size to be more informative when compared to the often arbitrary epochs.

Comment (ksalama) [reply cloudml-template/examples/babyweight-custom/trainer/task.py]: @dizcology - Thank you for the comment. I tried to give two options for the user the control the total number of iterations:
1. via explicitly specifying **train_steps** (in which case, num_epochs is irrelevant). This option is probably more relevant to large training data size with unknown number of examples

2. via specifying both **num_epochs** and **train_size** (in which case, the train_steps will be (train-size/train-batch-size) * num-epochs. This option is probably more relevant to relatively small training data size with known number of examples (train_size)

However, if you think it is better to only have one way, happy to do this in the following PR :-)

**Comment by puneith** [comment]
@dizcology and others also have the permission to do that :-) But merging it now. @ksalama I noticed you were not in the collaborators for some reason and added you.

**Pull request merged**